drivers/gpu/drm/virtio/virtgpu_drv.h

```diff
@@ -244,6 +245,8 @@ struct virtio_gpu_device {
 	struct virtio_gpu_output outputs[VIRTIO_GPU_MAX_SCANOUTS];
 	uint32_t num_scanouts;
 	uint32_t num_vblankq;
+	uint32_t output_cap_mask;
```

Comment:
it is better to add some description for this new added member

Reply:
ok